blog/models.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=100)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_created=True)
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE,
+        related_name="comments"
+    )
+    post = models.ForeignKey(
+        Post,
+        on_delete=models.CASCADE,
+        related_name="post_comments"
```

Comment:
Delete the `post_` part from your related name, it would be over-explicit to call `Post.post_comments` to retrieve commentary objects.